### PR TITLE
Added crud functionality

### PR DESCRIPTION
**Pull request by AinsleyB29**
### Related issue(s): 
- #47 

### Description of changes made: 
- Added PhotoViewSet
- Added Photo Serializer
- Added main router url path

### (optional) Questions or comments for review:
- any additional questions or comments goes here.

### CODE REVIEW

File: back-end/photo_app/views.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+from rest_framework import viewsets, permissions
+
+from .serializers import PhotoSerializer
+from .models import Photo
+import json
+
+class PhotoViewSet(viewsets.ModelViewSet):
+    queryset = Photo.objects.all()
+    serializer_class = PhotoSerializer
+    permission_classes = [permissions.IsAuthenticatedOrReadOnly]
```
Comment (Underwaterr) [inline back-end/photo_app/views.py:10]: Let's disable authentication for now

File: back-end/photo_app/views.py [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+
+
+
+
```
Comment (Underwaterr) [inline back-end/photo_app/views.py:14]: lots of whitespace 👀

**Review by Underwaterr — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "slothstagram",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```
Comment (Underwaterr) [inline package-lock.json:6]: Why is this here?

File: back-end/photo_app/admin.py [hunk 4]
```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
+from .models import Photo
+
+admin.site.register(Photo)
```
Comment (Underwaterr) [inline back-end/photo_app/admin.py:4]: I still don't see the Photos in the admin panel, make sure you aren't missing any other steps ;)

**Review by Underwaterr — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 3 above)
Comment (AinsleyB29) [reply package-lock.json]: will remove

**Review by Underwaterr — APPROVED** [review]
Radical!

**Review by Underwaterr — CHANGES_REQUESTED** [review]
Fix merge conflict :O

File: back-end/photo_app/models.py [hunk 5]
```diff
@@ -1,5 +1,5 @@
 from django.db import models
-from django.contrib.auth import User
+from django.contrib.auth.models import User
```
Comment (Underwaterr) [inline back-end/photo_app/models.py:2]: You are importing the `User` model but it's not being used, plz remove!

File: back-end/photo_app/views.py [hunk 6]
```diff
@@ -1,3 +1,11 @@
-from django.shortcuts import render
+from rest_framework import viewsets
+
+from .serializers import PhotoSerializer
+from .models import Photo
+import json
```
Comment (Underwaterr) [inline back-end/photo_app/views.py:5]: You are importing `json` but also not using it, plz remove!

**Review by Underwaterr — CHANGES_REQUESTED** [review]
Close, please remove unnecessary imports!

**Review by Underwaterr — CHANGES_REQUESTED** [review]
fix errors

File: back-end/photo_app/urls.py [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+from django.urls import path, include
+from rest_framework import routers
+from .views import PhotoViewSet
+
+router = routers.DefaultRouter()
+router.register(r'user', PhotoViewSet)
+
+urlpatterns = [
+  path('', include(router.urls)),
+  path('admin/', admin.site.urls)
```
Comment (Underwaterr) [inline back-end/photo_app/urls.py:11]: The `admin/` route should be defined in the project-level `urls.py`, not within the `photo_app` app.

**Review by Underwaterr — CHANGES_REQUESTED** [review]
Almost

**Review by Underwaterr — APPROVED** [review]
totally slammin'
